constants.py

```diff
@@ -3,8 +3,20 @@
 """
 
 import os
+import thousandaire.trading_cost
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
 DATA_DIR = os.path.join(ROOT_DIR, 'data')
 DATA_LIST_ALL = ['currency_price_tw', 'workdays']
+ENCODE_NP = {
```

Comment:
This name is very confusing. Please name global variables according to their real world meanings instead of engineering meanings to make them intuitive and widely usable.
For example, `TRADING_INSTRUMENTS` may be better here.